etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/ruleengine/RuleEngineTest.groovy

```diff
@@ -39,6 +39,27 @@ class RuleEngineTest extends Specification {
         1 * mockRuleLoader.loadRules(_ as String) >> [Mock(Rule)]
     }
 
+    def "ensureRulesLoaded loads rules only once on multiple threads"() {
+        given:
+        def threadsNum = 2
```

Comment:
Let's up this to 10 so we have more threads in contention for the lock.  It's easy for 2 to not execute the method at the same time.

```suggestion
        def threadsNum = 10
```